docs/DeveloperGuide.md

```diff
@@ -346,14 +415,229 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **Extensions**
 
 * 2a. The list is empty.
+  * 2a1. ContactCS displays a message to the user saying that the list is empty
```

Comment:
Would it be better be put this as `1a` instead, since this should directly happen after step 1 instead of step 2